### PR TITLE
feat(cfw): add rule_hit_count attribute to cfw_protection_rule resource

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
add rule_hit_count attribute to cfw_protection_rule resource

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add rule_hit_count attribute to cfw_protection_rule resource
```

## PR Checklist

* [x] Tests passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cfw" TESTARGS="-run TestAccProtectionRule_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cfw -v -run TestAccProtectionRule_basic -timeout 360m -parallel 4
=== RUN   TestAccProtectionRule_basic
=== PAUSE TestAccProtectionRule_basic
=== CONT  TestAccProtectionRule_basic
--- PASS: TestAccProtectionRule_basic (166.84s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cfw166.908s

```

### CODE REVIEW

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_protection_rule.go [hunk 1]
```diff
@@ -470,6 +482,43 @@ func FilterRules(rules []interface{}, id string) (interface{}, error) {
...
+	if err != nil {
+		return nil, err
+	}
+
+	count, err := jmespath.Search("data.records[0].rule_hit_count", getProtectionRuleHitCountRespBody)
+	if err != nil {
+		return nil, err
+	}
+
+	return count, nil
```
Comment (houpeng80) [inline huaweicloud/services/cfw/resource_huaweicloud_cfw_protection_rule.go]:
```suggestion
	return  jmespath.Search("data.records[0].rule_hit_count", getProtectionRuleHitCountRespBody)
```

Comment (profoundwu) [reply huaweicloud/services/cfw/resource_huaweicloud_cfw_protection_rule.go]: fixed

**Pull request merged**